pkg/queue/stats_reporter_test.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package queue
+
+import (
+	"errors"
+	"testing"
+
+	"go.opencensus.io/stats/view"
+)
+
+const (
+	namespace = "default"
+	config    = "helloworld-go"
+	revision  = "helloworld-go-00001"
+)
+
+func TestNewStatsReporter_negative(t *testing.T) {
+	tests := []struct {
+		name      string
+		errorMsg  string
+		result    error
+		namespace string
+		config    string
+		revision  string
+	}{
+		{
+			"Empty_Namespace_Value",
+			"Expected namespace empty error",
+			errors.New("Namespace must not be empty"),
+			"",
+			config,
+			revision,
+		},
```

Comment:
Do you mind adding the keys for the values to the test-instances? Gives more inline context when scrolling through.